src/app.ts

```diff
@@ -0,0 +1,18 @@
+import { config } from "dotenv"
+import { DataSource } from "typeorm"
+import DigitsData from "./entities/DigitsData"
+
+config()
+
+export const MyDataSource = new DataSource({
+  type: "postgres",
+  host: process.env.DB_HOST,
+  port: parseInt(process.env.DB_PORT || "5432"),
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+  entities: [DigitsData],
+  synchronize: true,
```

Comment:
change to false, we dont know exactly how sync is working, and better control it with migraitons